gutenberg.php

```diff
@@ -86,7 +86,7 @@ function gutenberg_menu() {
  */
 function gutenberg_wordpress_version_notice() {
 	echo '<div class="error"><p>';
-	echo __( 'Gutenberg requires WordPress 4.8 or later to function properly. Please upgrade WordPress before activating Gutenberg.', 'gutenberg' );
+	echo __( 'Gutenberg requires WordPress 4.9 or later to function properly. Please upgrade WordPress before activating Gutenberg.', 'gutenberg' );
```

Comment:
so we want to bump the required version in the readme?

Reply:
We have to :)

Reply:
Yes, also for #4762 it seems.

Reply:
#4762 was closed instead of merged, but since this has been merged, it can be revisited.